doc/ref_impl/cntt-ri/chapters/chapter06.md

```diff
@@ -61,241 +63,400 @@ Thanks to the descriptor file, the NFVi infrastructure deployment could be compl
 Depends xxx.
 
 <a name="6.3"></a>
-## 6.3 Descriptor file definition
+## 6.3 Descriptor file definition specification
 There must be a Descriptor File definition, which used by installer as input of necessary configuration.
-Mandatory and optional definition shall be defined.
+Mandatory and optional definition shall be defined.  
+All the required definition for description file is listed in this specification session, there's no restrictions on how to use it,
+there could be multiple ways to implement PDF, the implementation will be in next session
 
 <a name="6.3.1"></a>
-### 6.3.1 Hardware resource information
-The support of different workload types, each with different compute, storage, network requirements which needs kinds of hardware configuration template, we name this as server template.
-Besides it may include optional information such area name, data center name etc.
+### 6.3.1 Resource Pool information
+This table is the description of the resource pool, it contains only 2 parameters: name and type of the resource pool.
+
+Only one instance per resource pool.
```

Comment:
Can we clarify "instance" ... e.g. A resource pool maps to only one instance of abc ?

Reply:
I wanted to say : value is unique for these parameters,  we do not need  multiple entries for these parameters.
so is "A resource pool maps to only one instance of below parameters." the correct expression?